### DISCUSSION TITLE
Trying to add SubMenu like League Info

### DISCUSSION THREAD

**Discussion by JimNayzium** [nmelhado/league-page#249] [Help!]
I found the tabs.js and was able to add links to the main navigation area. 
Then I tried to copy and paste the entire League Info tab with the children array so I could recreate one like that.
I was successful at that. I renamed it and entered the routes and see all that.
However, when I click the new link in the browser it opens the League Info tab only. Meaning my new one opens the League Info submenu and the League Info tab getting clicked opens the League Info submenu.

I found the NavLarge.svelte file and it appears it only creates ONE instance of one sub-menu maybe? 
I am very very very new to svelte in so maybe I missed something?
I assumed when I found the tabs.js I was home free but I guess there is more to add.
I played with the looping {each} on the html side of the NavLarge.svelte file and couldn't hone it in.

I also thought maybe there was an identifier I failed to add but I didnt' see on in the League Info area of the tabs.js. I assumed it would be a property under it but it wasn't.

How do I add a main-nav link with a submenu of links like League Info? 

THANKS for the great codebase to start with!

**Comment by efuentesjr** [comment]
@JimNayzium I've figured this out. I was able to add a Playoff Projection tab and I linked info into that. Let me know if you still need help here.

**Reply by JimNayzium** [reply]
Thanks so much for the reply. I have learned a LOT about sveltekit since first downloading all this. I think I got what I need now but I appreciate you!
